APSHTTPClient/APSHTTPRequest.m

```diff
@@ -542,14 +549,21 @@ - (void)connectionDidFinishLoading:(NSURLConnection *)connection
     self.response.readyState = APSHTTPResponseStateDone;
     self.response.connected = NO;
      
-    [self invokeCallbackWithState:APSHTTPCallbackStateReadyState];
+    [self responseFinished];
+}
 
+-(void)responseFinished
+{
+    [self invokeCallbackWithState:APSHTTPCallbackStateReadyState];
+    
     [self invokeCallbackWithState:APSHTTPCallbackStateSendStream];
-
+    
     [self invokeCallbackWithState:APSHTTPCallbackStateDataStream];
-
+    
     [self invokeCallbackWithState:APSHTTPCallbackStateLoad];
-
+    
+    NSURLCache *sharedCache = [[NSURLCache alloc] initWithMemoryCapacity:0 diskCapacity:0 diskPath:nil];
+    [NSURLCache setSharedURLCache:sharedCache];
```

Comment:
Not sure why I added that, thought it was somewhere before. Will remove it for now, since we already specify a cache policy.